src/Symfony/Component/HttpFoundation/File/MimeType/MimeTypeExtensionGuesser.php

```diff
@@ -728,6 +728,7 @@ class MimeTypeExtensionGuesser implements ExtensionGuesserInterface
         'text/tab-separated-values' => 'tsv',
         'text/troff' => 't',
         'text/turtle' => 'ttl',
+        'text/twig' => 'twig',
```

Comment:
That MIME type is not officially registered with IANA. I don't know what Symfony's general policy is here, but "inventing" an unofficial internal MIME type without any vendor prefixes might be dangerous.

Reply:
why is it dangerous? lol

Reply:
Because it can have unanticipated consequences :)